README.md

```diff
@@ -4,3 +4,10 @@ This repo is a collection of code supporting the Wiki Loves Monuments competitio
 
 At the moment it mostly consists of bots that analyze and prepare the monument pages. 
 See the [bots README](update-bot/README.md) for detailed documentation on the bots.
+
+## Installation
+
+Python 2
+
+    python -m pip install -r requirements.txt
+    python -m pip install -r dev-requirements.txt
```

Comment:
Because my pip is broken in some weird way and only the long version works. Putting this in here so I can remember the long version :)